server/src/main/java/io/deephaven/server/session/SessionService.java

```diff
@@ -85,7 +85,7 @@ public StatusRuntimeException transform(final Throwable err) {
                 } else if (sre.getStatus().getCode().equals(Status.CANCELLED.getCode())) {
                     log.debug().append("ignoring cancelled request").endl();
                 } else {
-                    log.error().append(sre).endl();
+                    log.debug().append(sre).endl();
```

Comment:
As discussed in the meeting yesterday; this doesn't affect DHE as DHE has its own error transformer. This change only swallows SREs which are the intended way to communicate expected-kinds of errors to clients. 

In the context of a dashboard, it should be the dashboard client that raises the alarm of a missing PQ table - so that feels appropriate .

Reply:
In the dashboard context, the client is your web browser.  The server should have a log as well, otherwise the developer does not have information to debug it.  In the DHE context, an info/warn instead of error message could be appropriate.  If this part of the SessionService is not used by DHE that's fine.